Cargo.toml

```diff
@@ -1,27 +1,19 @@
 [package]
-name = "multipart"
-
-version = "0.16.1"
-
-authors = ["Austin Bonander <austin.bonander@gmail.com>"]
-
+name = "qiniu-multipart"
+version = "0.16.2"
+authors = ["Austin Bonander <austin.bonander@gmail.com>", "Rong Zhou <zhourong@qiniu.com>"]
```

Comment:
I can no longer merge this branch because of these changes. You probably want to check-out a new branch at the current commit and reset `master` to the previous commit and then force-push.

Reply:
Just for just notice that, I have reset `master` back.